pkg/virtctl/vm/vm.go

```diff
@@ -127,6 +127,7 @@ func NewMigrateCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 			return c.Run(args)
 		},
 	}
+	cmd.Flags().BoolVar(&dryRun, "dry-run", false, "--dry-run=false: Flag used to set whether to perform a dry run or not. If true the command will be executed without performing any changes.")
```

Comment:
I see this string is (`""--dry-run=false: Fla..."`) is duplicated many times in this file, can we save it as a const and re-use it?

Reply:
Totally agree! I noticed and reworked it in the current task I'm working. I will put that here. Thanks